### ISSUE TITLE
install script doesn't work on windows

### ISSUE THREAD

**Issue by Zvika88** [flovilmart/parse-image#5]
it is trying to run install.sh which doesn't work on windows and prevents the package from being installed

**Comment by flovilmart** [comment]
yeah that's problematic for now it's optimized for debian, ubuntu and MacOS. I'll try to think of something.

Can you provide the log please?

**Comment by pseudozach** [comment]
same issue on azure web app since it's a windows machine. here's the log:
'
016-03-04T07:03:54    > parse-image@0.1.1 preinstall D:\home\site\wwwroot\node_modules.staging\parse-image-66992af421fb593da9acf80555618bbc
2016-03-04T07:03:54    > ./install.sh
2016-03-04T07:03:54  
2016-03-04T07:03:54    '.' is not recognized as an internal or external command,
2016-03-04T07:03:54    operable program or batch file.
2016-03-04T07:03:55    npm ERR! Windows_NT 6.2.9200
2016-03-04T07:03:55    npm ERR! argv "D:\Program Files (x86)\nodejs\4.2.3\node.exe" "D:\Program Files (x86)\npm\3.5.1\node_modules\npm\bin\npm-cli.js" "install" "--production"
2016-03-04T07:03:55    npm ERR! node v4.2.3
2016-03-04T07:03:55    Failed exitCode=1, command="D:\Program Files (x86)\nodejs\4.2.3\node.exe" "D:\Program Files (x86)\npm\3.5.1\node_modules\npm\bin\npm-cli.js" install --production
2016-03-04T07:03:55    npm ERR! npm  v3.5.1
2016-03-04T07:03:55    An error has occurred during web site deployment.
2016-03-04T07:03:55    npm ERR! code ELIFECYCLE
2016-03-04T07:03:55  
2016-03-04T07:03:55    npm ERR! parse-image@0.1.1 preinstall: `./install.sh`
2016-03-04T07:03:55    npm ERR! Exit status 1
2016-03-04T07:03:55    npm ERR! 
2016-03-04T07:03:55    npm ERR! Failed at the parse-image@0.1.1 preinstall script './install.sh'.
2016-03-04T07:03:55    npm ERR! Make sure you have the latest version of node.js and npm installed.
2016-03-04T07:03:55    npm ERR! If you do, this is most likely a problem with the parse-image package,
2016-03-04T07:03:55    npm ERR! not with npm itself.
2016-03-04T07:03:55    npm ERR! Tell the author that this fails on your system:
2016-03-04T07:03:55    npm ERR!     ./install.sh
2016-03-04T07:03:55    npm ERR! You can get information on how to open an issue for this project with:
2016-03-04T07:03:55    npm ERR!     npm bugs parse-image
2016-03-04T07:03:55    npm ERR! Or if that isn't available, you can get their info via:
2016-03-04T07:03:55    npm ERR!     npm owner ls parse-image
2016-03-04T07:03:55    npm ERR! There is likely additional logging output above.
'

**Comment by safran1** [comment]
same problem

**Comment by Code-Crash** [comment]
Same here, not working on windows, any update on this, please?
